packages/cli/internal/pkg/cli/context/info_test.go

```diff
@@ -25,3 +27,33 @@ func TestDetail_IsNotEmpty(t *testing.T) {
 	detail := Detail{WesUrl: "amazon.com"}
 	assert.False(t, detail.IsEmpty())
 }
+
+func TestSummary_IsHeadProcessEngine(t *testing.T) {
```

Comment:
Could you please make your test consistent with other tests?

Here is a good example that I would suggest to follow:
https://github.com/aws/amazon-genomics-cli/blob/2a8fab00ceedfd72321f5466183edf92f4c4d0fa/packages/cli/internal/pkg/cli/logs_workflow_test.go#L65